index.html

```diff
@@ -1,12 +1,216 @@
+              <img src="./assets/img/lecture-icon.png" alt="lecture icon"/>
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Lecture</h3>
+              <p class="custom-white text-10 big-text-12">
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/exhibition-icon.png" alt="exhibition icon"/>
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">CC Exhibition</h3>
+              <p class="custom-white text-10 big-text-12">
+                Appreciate various creations applying CC license of artists,
+                organized from Art Center Nabi
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/forum-icon.png" alt="forum icon"/>
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Forum</h3>
+              <p class="custom-white text-10 big-text-12">
+                Have the time to share your thoughts and opinions with experts
+                for each topic.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/workshop-icon.png" alt="workshop icon"/>
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Workshop</h3>
+              <p class="custom-white text-10 big-text-12">
+                Try creating your own work using open source license rather than
+                just watching at it.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/ignite-icon.png" alt="ignite icon"/>
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">CC Ignite</h3>
+              <p class="custom-white text-10 big-text-12">
+                get opportunities to network with CC affiliates around the
+                world, also after the summit.
+              </p>
+            </div>
+          </div>
+        </div>
+        <button class="action-btn-2 flex-row custom-white text-16">
+          Join WD Global Summit 2023
+        </button>
+        <a class="btn-alt custom-white text-20-B" href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+    <!-- Speaker section-->
+    <section class="speakers-container flex-column">
+      <div class="speakers-content"></div>
+    </section>
```

Comment:
- [ ] It seems like the changes you made to this section broke your images. Currently, the speaker images are not displaying, please you need to fix this as all the images are meant to display properly on your website. :smile: :+1: 

- SCREENSHOT FOR REFERNCE:
![Selection_018](https://user-images.githubusercontent.com/107632780/234997795-e4d16e13-7762-4730-8db4-9b16a9d6f4f4.png)